src/index.ts

```diff
@@ -95,29 +101,14 @@ export default class QuickbooksConnector extends BaseHttpConnector<
   }
 
   private async storeQBToken(wrapper: QuickBookTokenWrapper): Promise<any> {
-    const newToken = {
-      realmID: this.configOptions.realmId,
-      token: wrapper.token,
-      access_expire_timestamp: wrapper.access_expire_timestamp,
-      refresh_expire_timestamp: wrapper.refresh_expire_timestamp,
-    }
-    const dbToken = await this.app.getPersistentStore().set(this.getTokenKey(), newToken)
+    const dbToken = await this.app.getPersistentStore().set(this.getTokenKey(), wrapper)
```

Comment:
Could just `return` here?

Reply:
This has been addressed on the `storeQBToken` function - but not on the `getQBToken` one